configuration/appframework/scripts/patientHasEncounterOfTypeDuringProgramEnrollment.js

```diff
@@ -0,0 +1,12 @@
+function patientHasEncounterOfTypeDuringProgramEnrollment(programs, encounterTypeUuid, programUuid) {
+
+  var patientEnrolled = isPatientEnrolled(programs, programUuid);
+  if ( !patientEnrolled ) {
+    return false;
+  }
+  var dateEnrolled = getDateEnrolledByProgramUuid(programs, programUuid);
+  if ( !dateEnrolled ) {
+    return false;
+  }
```

Comment:
I don't see the point in checking both of these things above.  All you need is the `dateEnrolled`.  No point in checking `isPatientEnrolled` when a null `dateEnrolled` gives you the same information.